### PR TITLE
[docs] Changing the path in examples for modules

### PR DESCRIPTION
**Pull request by unreturned**
## Description
Changing the path in examples for modules for consistent with previous private registry path (`corp.company.com:5000/deckhouse/modules` -> `corp.company.com:5000/sys/deckhouse/modules`).
Fix helper script (variable not passed to `<<EOF` block)

## Changelog entries

```changes
section: docs
type: chore
summary: |
  Changing the path in examples for modules for consistent with previous private registry path.
  Fix helper script.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 1]
```diff
@@ -616,11 +617,11 @@ Follow these steps for manual loading images of modules, connected from the modu
 
    ```shell
    d8 mirror modules push \
-     -d /tmp/d8-modules --registry='corp.company.com:5000/deckhouse/modules' \
+     -d /tmp/d8-modules --registry='corp.company.com:5000/sys/deckhouse/modules' \
      --registry-login='<USER>' --registry-password='<PASSWORD>'
    ```
 
-   > Before pushing images, make sure that the path for loading into the registry exists (`/deckhouse/modules` in the example above), and the account being used has write permissions.
+   > Before pushing images, make sure that the path for loading into the registry exists (`/sys/deckhouse/modules` in the example above), and the account being used has write permissions.
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:624]:
```suggestion
   > Before images, make sure that the path for loading into the registry exists (`/sys/deckhouse/modules` in the example above), and the account being used has write permissions.
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 2]
```diff
@@ -638,7 +639,7 @@ Follow these steps for manual loading images of modules, connected from the modu
      registry:
        # Specify the authentication string for your registry.
        dockerCfg: <BASE64_REGISTRY_CREDENTIALS>
-       repo: 'corp.company.com:5000/deckhouse/modules'
+       repo: 'corp.company.com:5000/sys/deckhouse/modules'
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:642]:
```suggestion
       repo: 'corp.company.com:5000/sysололдолдdeckhouse/modules'
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9693](https://github.com/deckhouse/deckhouse/pull/9693) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**